introducing-java/src/main/java/com/onto/javacourse/intro/gcd/GCD.java

```diff
@@ -1,5 +1,33 @@
-public class Main {
-    public static void main(String[] args) {
+public class GCD {
 
+    /**
+     * Method to find GCD
+     */
+    public static int GCD(int a, int b) {
```

Comment:
a and b are bad for parameter names. Better would be to have something like firstNumber, secondNumber